### ISSUE TITLE
MatSelect Does not voice currently selected value with a Screen Reader 

### ISSUE THREAD

**Issue by JeffKochuk** [angular/components#12297]
#### Bug, feature request, or proposal:
When tabbing to focus a mat-select component, the currently-selected value is not voiced.

#### What is the expected behavior?
The display value of the currently selected value is voiced when the 

#### What is the current behavior?
Screenreader  says "List Box" and any labelling information, but not the currently selected value.

#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>
https://stackblitz.com/edit/angular-material2-issue-cvrarp

Enable Screen Reader
Tab to the mat-select
Screenreader says "Listbox"
Press Space
Screenreader announces the options.
Press Enter
Screenreader does not announce any changes or say what was selected.

#### What is the use-case or motivation for changing an existing behavior?
Trying to get GAR 4 in my app. 

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Chrome + ChromeVox

**Comment by mtotheoneandonlyk** [comment]
Any progress here? I still having the problem with Jaws screen reader.

**Comment by qrdean** [comment]
Any update on when this will be implemented? This is still broken for us. 

We ended up switching to native select. (Yay, that was added in v7!) Unfortunately that was not a simple switch, but it was necessary to get it across the line.

**Comment by Mr1upMachine** [comment]
@jelbourn or @crisbeto Is there any progress on this issue?

**Comment by jelbourn** [comment]
I believe this should have been fixed by #20082
